### ISSUE TITLE
Text does not have sufficient contrast; impact on A11Y

### ISSUE THREAD

**Issue by tripu** [haplo/Flex#28]
Nice project, @haplo!

Looking at the demo page, I noticed captions and subtitles are a bit hard to read for me (my eyesight isn't good). I suspected contrast ratio is not high enough.

[A quick analysis seems to confirm that](https://pagespeed.web.dev/analysis/https-haplo-github-io-pelican-theme-reflex/gwzby6fjxk?form_factor=mobile) (navigate to _accessibility_ &rarr; _contrast_, then expand section _background and foreground colours do not have a sufficient contrast ratio_).
